lib/components/dropdown.vue

```diff
@@ -43,15 +43,6 @@
     </div>
 </template>
 
-<style scoped>
-    .dropdown-item:focus,
-    .dropdown-item:hover,
-    .dropdown-header:focus {
-        background-color: #eaeaea;
```

Comment:
For table we created a `.bv-table` custom class to handle some specific styling issues.